packages/suite-web/e2e/tests/metadata/wallet-metadata.test.ts

```diff
@@ -3,21 +3,17 @@
 
 import { rerouteMetadataToMockProvider, stubOpen } from '../../stubs/metadata';
 
-const firmwares = [
-    '2.2.0',
-    // , '2-main'
-] as const;
+const firmwares = ['2.2.0', , '2-main'] as const;
 const provider = 'dropbox';
 
 const mnemonic = 'all all all all all all all all all all all all';
 // state corresponding to all seed
-const standardWalletState = 'mvbu1Gdy8SUjTenqerxUaZyYjmveZvt33q@355C817510C0EABF2F147145:0';
+const standardWalletState = 'mvbu1Gdy8SUjTenqerxUaZyYjmveZvt33q@355C817510C0EABF2F147145:1';
 // state corresponding to "wallet for drugs"
-const firstHiddenWalletState = 'myBrmyzvN5Wa4oeYrL7t8EYU1Ch5Q6vp47@355C817510C0EABF2F147145:1';
+const firstHiddenWalletState = 'myBrmyzvN5Wa4oeYrL7t8EYU1Ch5Q6vp47@355C817510C0EABF2F147145:2';
 // state corresponding to "C"
-const secondHiddenWalletState = 'mkx2Uqi3fmLHh8AHpQvAErTM3MZpzrmFr2@355C817510C0EABF2F147145:2';
-
-describe('Metadata - wallet labeling', () => {
+const secondHiddenWalletState = 'mkx2Uqi3fmLHh8AHpQvAErTM3MZpzrmFr2@355C817510C0EABF2F147145:3';
```

Comment:
interesting. I don't remember changing this in connect and if I remember correctly these are values returned from connect 🤔